apollo-federation/src/sources/connect/expand/mod.rs

```diff
@@ -614,6 +614,32 @@ mod helpers {
                         ))
                     }
                 }?;
+            } else {
```

Comment:
@nicholascioli let's chat about this when you get a second — this is a bit hacky and there might be another way